docs/content/envoy-external-authorization.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Envoy External Authorization
+kind: tutorial
+weight: 8
+---
+
+[Envoy](https://www.envoyproxy.io/docs/envoy/v1.10.0/intro/what_is_envoy) is a L7 proxy and communication bus designed for large modern service oriented architectures. Envoy (v1.7.0+) supports an [External Authorization filter](https://www.envoyproxy.io/docs/envoy/v1.10.0/intro/arch_overview/ext_authz_filter.html) which calls an authorization service to check if the incoming request is authorized or not.
+
+This feature makes it possible to delegate authorization decisions to an external service and also makes the request context available to the service which can then be used to make an informed decision about the fate of the incoming request received by Envoy.
+
+## Goals
+
+The tutorial shows how Envoy’s External authorization filter can be used with OPA as an authorization service to enforce custom security policies over API requests received by Envoy.
```

Comment:
Let's mention that the tutorial shows how to write enforce policies over the request body.